src/Web/Sfa.Das.Sas.Shared.Components/Controllers/BasketController.cs

```diff
@@ -73,11 +73,10 @@ public async Task<IActionResult> AddProviderFromDetails(SaveBasketFromProviderDe
         [HttpPost]
         public async Task<IActionResult> AddProviderFromResults(SaveBasketFromProviderSearchViewModel queryModel)
         {
-            if (!await IsInBasket(queryModel.SearchQuery.ApprenticeshipId, queryModel.ItemId))
+            if (!await IsInBasket(queryModel.SearchQuery.ApprenticeshipId, queryModel.Ukprn,queryModel.LocationId))
```

Comment:
Does `AddProviderFromDetails` also need locationId adding in to it's `IsInBasket()` check?

Reply:
yep, i'll add it in. good spot!